src/monaco-utils.ts

```diff
@@ -63,10 +63,14 @@ export class MonacoUtils {
     while (item = nav.next()) {
       elements.push(item);
     }
+
+    const AUTO_EXPAND_DIRECTORIES = ["assembly", "src"]
+    
     if (elements.length != 0){
       for (let i = 0, len = elements.length; i < len; i++) {
-        if (elements[i].element.name == "assembly" || elements[i].element.name == "src")
-        model.expand(elements[i]);
+        if (elements[i].element.name == AUTO_EXPAND_DIRECTORIES[0] || elements[i].element.name == AUTO_EXPAND_DIRECTORIES[1]){
```

Comment:
any reason why hard code for 2 elements in array instead of suggested?

```
if (AUTO_EXPAND_DIRECTORIES.indexOf(elements[i].element.name) != -1) {
    model.expand(elements[i]);
}
```